lib/kube/proxy/moderated_sessions_test.go

```diff
@@ -429,3 +450,20 @@ func TestModeratedSessions(t *testing.T) {
 		})
 	}
 }
+
+// validateSessionTracker validates that the session tracker has the expected
+// reason and invited users.
+func validateSessionTracker(t *testing.T, testCtx *TestContext, sessionID string, reason string, invited []string) error {
```

Comment:
Drop the `t` param? I don't think we are using it. Or, alternative, drop the error response and use it?